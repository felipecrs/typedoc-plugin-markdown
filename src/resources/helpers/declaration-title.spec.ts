import * as Handlebars from 'handlebars';

import { setup } from '../../../test/utils';

describe(`[helpers] declarationTitle`, () => {
  let project: any;

  beforeAll(() => {
    ({ project } = setup());
  });

  test(`should compile`, () => {
    expect(Handlebars.helpers.declarationTitle.call(project.findReflectionByName('color'))).toMatchSnapshot();
  });
});
