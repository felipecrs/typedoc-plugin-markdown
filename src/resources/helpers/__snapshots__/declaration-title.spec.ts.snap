// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[helpers] declarationTitle should compile 1`] = `"**color**: *string*"`;
